tests/test_physics.py

```diff
@@ -189,5 +195,96 @@ def test_dynamics(sim):
         obs = sim.step(random.choice(list(hab_cfg.agents[0].action_space.keys())))
 
         # 2nd object should no longer rotate or translate
-        assert previous_object_states[1][0] == sim.get_translation(object2_id)
+        assert np.allclose(
+            previous_object_states[1][0], sim.get_translation(object2_id)
+        )
         assert previous_object_states[1][1] == sim.get_rotation(object2_id)
+
+        # test modifying gravity
+        new_object_start = np.array([100.0, 0, 0])
+        sim.set_translation(new_object_start, object_id)
+        new_grav = np.array([10.0, 0, 0])
+        sim.set_gravity(new_grav)
+        assert np.allclose(sim.get_gravity(), new_grav)
+        assert np.allclose(sim.get_translation(object_id), new_object_start)
+        sim.step_physics(0.1)
+        assert sim.get_translation(object_id)[0] > new_object_start[0]
+
+
+def test_velocity_control(sim):
+    cfg_settings = examples.settings.default_sim_settings.copy()
+    cfg_settings["scene"] = "NONE"
+    cfg_settings["enable_physics"] = True
+    hab_cfg = examples.settings.make_cfg(cfg_settings)
+    sim.reconfigure(hab_cfg)
+    sim.set_gravity(np.array([0, 0, 0.0]))
+
+    sphere_path = osp.abspath("data/test_assets/objects/sphere")
+    template_ids = sim.load_object_configs(sphere_path)
+    object_template = sim.get_object_template(template_ids[0])
```

Comment:
Do I need to load template if I use `MotionType.KINEMATIC`?

I am trying to use velocity control with the agent.